__tests__/app.test.js

```diff
@@ -93,4 +94,31 @@ describe('GET /api/articles/:article_id', () => {
                 expect(body.msg).toBe('Invalid article ID');
             });
     });
+});
+
+describe('GET /api/articles', () => {
+    test('responds with status 200 and an array of articles sorted by date', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({ body }) => {
+                const { articles } = body;
+                expect(articles).toBeInstanceOf(Array);
+                expect(articles.length).toBeGreaterThan(0);
+
+                articles.forEach((article) => {
```

Comment:
Very clear - if you want to stretch your knowledge of jest take a look at the `.toMatchObject` matcher and try to refactor this test to apply your learning.